workflows.md

```diff
@@ -48,24 +48,24 @@ on:
 jobs:
   check-test:
     name: Check 📦
-    uses: boehringer-ingelheim/dv.templates/.github/workflows/check.yml@main
+    uses: boehringer-ingelheim/dv.templates/.github/workflows/check.yml@local
 
   lintr:
     name: Lintr 🔍
-    uses: boehringer-ingelheim/dv.templates/.github/workflows/lintr.yml@main
+    uses: boehringer-ingelheim/dv.templates/.github/workflows/lintr.yml@local
     with:
       lintr_error_on_lint: true
 
   gitleaks:
     name: Gitleaks 🌧️
-    uses: boehringer-ingelheim/dv.templates/.github/workflows/gitleaks.yml@main
+    uses: boehringer-ingelheim/dv.templates/.github/workflows/gitleaks.yml@local
```

Comment:
this should still be @main before merging

Reply:
solved in 8655fb0f251729d017ec1bf9c8eb6b72f3246d2c